aries_cloudagent/protocols/coordinate_mediation/v1_0/manager.py

```diff
@@ -151,6 +189,11 @@ async def get_keylist(self, record: MediationRecord) -> Sequence[RouteRecord]:
         route_mgr = RoutingManager(self.context)
         return await route_mgr.get_routes(record.connection_id)
 
+    async def create_keylist(self, record: MediationRecord, did: DIDInfo) -> RouteRecord:
+        """Retrieve routes for connection."""
+        route_mgr = RoutingManager(self.context)
+        return await route_mgr.create_route_record(record.connection_id, did.verkey)
+
```

Comment:
I'm suspicious of this method. Might just be the incorrect doc string.

Reply:
"kinda sus!" I will update the document string, but this method may not be needed anymore. should we remove it?